addons/web/controllers/webmanifest.py

```diff
@@ -95,3 +100,117 @@ def offline(self):
         return request.render('web.webclient_offline', {
             'odoo_icon': base64.b64encode(file_open(self._icon_path(), 'rb').read())
         })
+
+    @http.route('/scoped_app', type='http', auth='public', methods=['GET'])
+    def scoped_app(self, app_id, path='', app_name=''):
+        """ Returns the app shortcut page to install the app given in parameters """
+        _app_name = unquote(app_name) if app_name else self._get_scoped_app_name(app_id)
+        _path = f"/{unquote(path)}"
```

Comment:
I would replace to two variables in place, I don't think it is worth to keep the quoted values around